packages/components/Menu/src/MenuGroup/MenuGroup.tsx

```diff
@@ -0,0 +1,44 @@
+/** @jsx withSlots */
+import React from 'react';
+import { View } from 'react-native';
+
+import { compose, withSlots } from '@fluentui-react-native/framework';
+import type { UseSlots } from '@fluentui-react-native/framework';
+
+import { stylingSettings } from './MenuGroup.styling';
+import type { MenuGroupProps, MenuGroupType } from './MenuGroup.types';
+import { menuGroupName } from './MenuGroup.types';
+
+export const MenuGroup = compose<MenuGroupType>({
+  displayName: menuGroupName,
+  ...stylingSettings,
+  slots: {
+    root: View,
+  },
+  useRender: (userProps: MenuGroupProps, useSlots: UseSlots<MenuGroupType>) => {
+    const Slots = useSlots(userProps);
+    return (_final: MenuGroupProps, children: React.ReactNode) => {
+      let itemPosition = 0;
+      const childrenWithSet = React.Children.toArray(children).map((child) => {
+        if (React.isValidElement(child)) {
+          const itemCount = React.Children.toArray(children).filter(
+            (child) => React.isValidElement(child) && (child as any).type.displayName !== 'MenuGroupHeader',
+          ).length;
+
+          if ((child as any).type.displayName !== 'MenuGroupHeader') {
+            itemPosition++;
+          }
+          return React.cloneElement(
+            child as React.ReactElement<unknown, string | React.JSXElementConstructor<any>>,
+            {
+              accessibilityPositionInSet: child.props.accessibilityPositionInSet ?? itemPosition, // win32
+              accessibilitySetSize: child.props.accessibilitySetSize ?? itemCount, //win32
+            } as any,
+          );
+        }
+        return child;
+      });
+      return <Slots.root>{childrenWithSet}</Slots.root>;
+    };
+  },
+});
```

Comment:
I'm not sure where the issue lies but n-of-m properties aren't being set on submenu triggers on win32. I'll continue digging but the issue doesn't appear in the main branch.

Reply:
Oh sorry - this is because your branch doesn't include the latest changes. I merged in the code that enabled n-of-m for submenu triggers last week so it should be there once you merge them in from main.